### PR TITLE
feat: add codeclimate config and .eslintrc

### LINKED ISSUE #43: Add CodeClimate Config
I've added this repo to code climate. This project should pull in a code climate config from dashboard and generate code coverage reports from CircleCI

### LINKED ISSUE #57: Fix "react/sort-prop-types" issue in src/components/Breadcrumbs/Breadcrumbs.js
Prop types declarations should be sorted alphabetically

https://codeclimate.com/github/DecipherNow/gm-ui-components/src/components/Breadcrumbs/Breadcrumbs.js#issue_5a67ac05505ce80001000103

### LINKED ISSUE #58: Fix "react/prop-types" issue in src/components/Breadcrumbs/BreadcrumbItem.js
'expand' is missing in props validation

https://codeclimate.com/github/DecipherNow/gm-ui-components/src/components/Breadcrumbs/BreadcrumbItem.js#issue_5a67ac04505ce80001000102

### LINKED ISSUE #59: Fix "react/sort-prop-types" issue in src/components/Checkbox/Checkbox.js
Prop types declarations should be sorted alphabetically

https://codeclimate.com/github/DecipherNow/gm-ui-components/src/components/Checkbox/Checkbox.js#issue_5a67ac04505ce80001000101

### LINKED ISSUE #60: Fix "react/sort-prop-types" issue in src/components/Icon/Icon.js
Prop types declarations should be sorted alphabetically

https://codeclimate.com/github/DecipherNow/gm-ui-components/src/components/Icon/Icon.js#issue_5a67ac07505ce80001000104

### PR DESCRIPTION
**Pull request by kaitmore**
Resolves #43 
Resolves #57 
Resolves #58 
Resolves #59  
Resolves #60

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "parserOptions": {
+    "ecmaVersion": 2016,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "experimentalObjectRestSpread": true,
+      "modules": true
+    }
+  },
+  "plugins": ["react", "prettier"]
```
Comment (spmcbride1201) [inline .eslintrc]: You need to install these plugins

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "react/sort-comp": 1,
+    "react/sort-prop-types": 1,
+    "react/wrap-multilines": 0,
+    "jsx-a11y/href-no-hash": "off"
+  },
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "extends": ["react-app", "prettier", "prettier/react"],
```
Comment (spmcbride1201) [inline .eslintrc]: You need to install these eslint configs

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,60 @@
+{
+  "parser": "babel-eslint",
```
Comment (spmcbride1201) [inline .eslintrc]: You need to install eslint

**Comment by kaitmore** [comment]
@spmcbride1201 installed plugins/configs, adding a linting script, and confirmed linting works locally

**Pull request merged**